bodhi/server/models.py

```diff
@@ -2965,12 +2964,10 @@ def comment(self, session, text, karma=0, author=None, anonymous=False,
                 caveats.append({'name': 'karma', 'description': notice})
 
         comment = Comment(
-            text=text, anonymous=anonymous,
+            text=text, anonymous=False,
```

Comment:
To answer one of your questions, I think we can drop this field from the class. We do want to keep the historical data about which comments were anonymous, but as you pointed out that data is also present in the username which I think is good enough.